cl/api/templates/rest-docs-vlatest.html

```diff
@@ -13,8 +13,29 @@
 {% endblock %}
 
 {% block content %}
-  <div class="col-sm-7">
+  <div class="col-sm-3">
+
+    {% include "includes/toc_sidebar.html" %}
+
+  </div>
+  <div class="col-sm-9">
     <h1 id="about">REST API &ndash; v3.14</h1>
+
+    <div class="well">
+      <h3 class="top">Your Authorization Token</h3>
+      {% if user.is_authenticated %}
+        <p>Your permanent token for use with this API is:</p>
+        <h4><code>{{ user.auth_token }}</code></h4>
+        <p>Do not share your API token. Keep it private like a password.
+        </p>
+        <p class="text-right"><a class="btn btn-default" href="{% url "view_api" %}#usage">See Usage</a></p>
+      {% else %}
+        <p><a href="{% url "sign-in" %}?next={{ request.path }}"
+                class="btn btn-primary btn-lg">Sign In To See Your Token</a>
+        </p>
+      {% endif %}
+    </div>
```

Comment:
Maybe this goes better in the authentication section?

Reply:
Done.